### ISSUE TITLE
Error on startup when u try to join

### ISSUE THREAD

**Issue by BroGoDx** [BlackDemonZyT/BotSentry#101]
When server is starting up and u try to join there is an error
https://pastebin.com/LtF8fDmm
ImanitySpigot (1.8.8 fork) 
Java 17 
botsentry 9.5.3

The error doesn't happen if nobody joins and no errors at all
https://pastebin.com/gZwfEjLU

**Comment by Laurenshup** [comment]
Do you mean, while trying to join when starting up?
You are not using /reload, correct?

**Comment by BroGoDx** [comment]
Yes sorry for bad English. No /reload. Restarting, starting when server is stopped yes,

**Comment by Laurenshup** [comment]
That's odd, since joining would not have any effect normally. What plugins are you running? What versions?
